.github/ISSUE_TEMPLATE/⚙️-submit-a-new-resource.md

```diff
@@ -1,25 +1,20 @@
 ---
+order: 3
 name: "⚙️ Submit a new resource"
 about: 'Have a great resource you''d like to add to the README? '
 title: "[Resource]"
 labels: awesome-readme
 assignees: nitya, savannahostrowski
 
 ---
-
-## New resource to add to the README
-
 ## 1. Name of resource
-_A brief name to provide context_.
 
 ## 2. Resource author
-_Who created this resource (so we can credit them!)?_.
+_Who created this resource?_.
```

Comment:
Might be good to clarify why this info is needed and where it'll appear on the awesome-azd site